_sources/governance.rst.txt

```diff
@@ -0,0 +1,169 @@
+
+The Project is developed by a team of distributed developers, called Contributors. Contributors are individuals who have contributed code, documentation, designs or other work to The Project. Anyone can be a Contributor. Contributors can be affiliated with any legal entity or none. Contributors participate in the project by submitting, reviewing and discussing GitHub Pull Requests and Issues and participating in open and public Project discussions on GitHub, Gitter, and other channels. The foundation of Project participation is openness and transparency.
+
+The Project Community consists of all Contributors and Users of The Project. Contributors work on behalf of and are responsible to the larger Project Community and we strive to keep the barrier between Contributors and Users as low as possible.
+
+The Project is not a legal entity, nor does it currently have any formal relationships with legal entities.
+
+
+Governance
+----------
+This section describes the governance and leadership model of The Project.
+
+The foundations of Project governance are:
+
+* Openness & Transparency
+* Active Contribution
+* Institutional Neutrality
+
+Traditionally, Project leadership was provided by a BDFL (Sergio Rey) and a subset of Contributors and Sub-Package Maintainers, called Core Developers, whose active and consistent contributions have been recognized by their receiving “commit rights” to the Project GitHub repositories. In general all Project decisions are made through consensus among the Core Developers with input from the Community.
+
+While this approach has served us well, as the Project grows we see a need for a more formal governance model. Moving forward The Project leadership will consist of a BDFL and Steering Council, including but not limited to all Sub-Package Maintainers. We view this governance model as the formalization of what we are already doing, rather than a change in direction.
+
+
+BDFL
+----
+The Project will have a BDFL (Benevolent Dictator for Life), who is currently Sergio Rey. As Dictator, the BDFL has the authority to make all final decisions for The Project. As Benevolent, the BDFL, in practice chooses to defer that authority to the consensus of the community discussion channels and the Steering Council (see below). It is expected, and in the past has been the case, that the BDFL will only rarely assert his/her final authority. Because rarely used, we refer to BDFL’s final authority as a “special” or “overriding” vote. When it does occur, the BDFL override typically happens in situations where there is a deadlock in the Steering Council or if the Steering Council asks the BDFL to make a decision on a specific matter. To ensure the benevolence of the BDFL, The Project encourages others to fork the project if they disagree with the overall direction the BDFL is taking. The BDFL may delegate his/her authority on a particular decision or set of decisions to any other Council member at his/her discretion.
```

Comment:
~~ remove ~~ (Benevolent Dictator for Life)

Reply:
... his/her...
- maybe we should either use more inclusive gender pronouns or stick with "they"